cylc/flow/scripts/cylc.py

```diff
@@ -650,9 +650,9 @@ def main():
                 else:
                     command = cmd_args.pop(0)
 
+            # this is an alias to a command
             if command in ALIASES:
-                # this is an alias to a command
```

Comment:
Nitpick: I disagree with the move of the comment (but actually I don't think the comment adds anything anyway)